spec/binarySearchTree_test.js

```diff
@@ -0,0 +1,26 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import BinarySearchTree from '../src/binarySearchTree'
+
+chai.use(chaiChange)
+
+describe('BinarySearchTree', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(BinarySearchTree).to.be.a('function')
+  })
+
+  context('insert() & search()', () => {
```

Comment:
As per feedback from JRob, any time I see an 'and' it means that the test should probably be split into two tests. That is not the case here, however. I think you can omit the 'and searches for them' part because you are testing the insert method and could use several other methods to accomplish this.